elements/pfe-progress-steps/README.md

```diff
@@ -0,0 +1,99 @@
+# PatternFly Element | Progress stepper element
+A component that gives the user a visual representation of the current state of their progress through an application (typically a multistep form).
+
+## Usage
+Describe how best to use this web component along with best practices.
+
+```html
+  <pfe-progress-steps>
+    <pfe-progress-steps-item state="active" current>
+      <div slot="title">Current</div>
+      <a slot="link" href="#">View current step</a>
+    </pfe-progress-steps-item>
+    <pfe-progress-steps-item>
+      <div slot="title">Next</div>
+      <a slot="link" href="#">View next step</a>
```

Comment:
```suggestion
      <a slot="description" href="#">View next step</a>
```

Is the slot named link or description?

Reply:
It should be "link" I believe.  It's meant to give authors a way to turn the item into a clickable item.

Reply:
Could we look to the a tag for that?  Like if the description slot uses an a tag, make it a link, otherwise, leave it as text?